cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: SERVICE_KEYVAULT_NAME
+  description: The name of the Key Vault containing keys, secrets and certificates used to support services running on the cluster service.
+  value: "service-kv-aro-hcp-dev"
+- name: TENANT_ID
+  description: The Tenant ID of Key Vault.
+  value: "64dc69e4-d083-49fc-9569-ebece1dd1408"
+
 objects:
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: cs-keyvault
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${AZURE_CS_MI_CLIENT_ID}
+      cloudName: AzurePublicCloud
+      keyvaultName: ${SERVICE_KEYVAULT_NAME}
+      objects: |-
+        array:
+          - |
+            objectName: firstPartyMock
+            objectType: secret
+            objectAlias: "firstPartyMock"
```

Comment:
Been thinking about this, is it possible to parameterise it like the keyvault name? I assume for the different environments, the name isn't going to have 'mock' on them 🤔 wdyt?